lib/pages/chat/chat.dart

```diff
@@ -1748,10 +1750,23 @@ class ChatController extends State<Chat>
     if (allMembershipEvents || canRequestHistory) {
       try {
         await requestHistory(historyCount: _defaultEventCountDisplay)
-            .then((response) {
-          Logs().d(
+            .then((response) async {
+          Logs().v(
             'Chat::_tryRequestHistory():: Try request history success',
           );
+          if (allMembershipEvents) {
+            await requestHistory(
+              historyCount: _defaultEventCountDisplay,
+              filter: StateFilter(
+                lazyLoadMembers: true,
+                types: [
+                  EventTypes.Message,
+                  EventTypes.Encrypted,
+                  EventTypes.Sticker,
+                ],
```

Comment:
need a method for this list: `getEventTypeToFilterUnnecessaryEvent`

Reply:
Updated